src/core/qgsbrowserproxymodel.h

```diff
@@ -147,7 +147,7 @@ class CORE_EXPORT QgsBrowserProxyModel : public QSortFilterProxyModel
      * Sets a filter to hide data items based on on item's data item provider key.
```

Comment:
```suggestion
     * Sets a filter to hide data items based on item's data item provider key.
```
and is it **item**'s data **item** provider key?